docs/_documentations/installoncloud.md

```diff
@@ -138,18 +138,13 @@ Go to **View**>**Find Command…**>**Codewind: Build**.
 ## Updating the version
 Restart the Codewind workspace in Che. Che automatically pulls the newest version of Codewind and the Theia extension.
 
-## Adding additional rules to support Codewind ODO extension
-The Codewind ODO extension needs to add additional rules for accessing OpenShift resources:
-1. In your home directory, run the following command to clone the Codewind ODO extension repository:
-```
-git clone https://github.com/eclipse/codewind-odo-extension
-```
-2. Login to your OpenShift/OKD cluster
-3. Go into `~/codewind-odo-extension/odo-RBAC` then run the following commands to add additional rules:
-```
-kubectl apply -f codewind-odoclusterrole.yaml
-kubectl apply -f codewind-odoclusterrolebinding.yaml
-```
+# Codewind OpenShift Do (odo) extension 
+See the [`codewind-odo-extension` repository](https://github.com/eclipse/codewind-odo-extension) for the extension to Codewind that provides support for odo projects.
 
-# Codewind OpenShift Do (ODO) extension 
-Extension to Codewind providing support for OpenShift projects: https://github.com/eclipse/codewind-odo-extension
\ No newline at end of file
+## Adding rules to support the Codewind odo extension
+The Codewind odo extension needs additional rules for accessing OpenShift resources. Use the following commands to clone the `codewind-odo-extension` repository, create the ClusterRole with the required permissions, and bind that ClusterRole to the Che workspace service account:
+```
+git clone https://github.com/eclipse/codewind-odo-extension &&\
+    kubectl apply -f codewind-odoclusterrole.yaml &&\
+    kubectl apply -f codewind-odoclusterrolebinding.yaml
+```
```

Comment:
Change the group of commands to this one
```
git clone https://github.com/eclipse/codewind-odo-extension &&\
   cd ./codewind-odo-extension/odo-RBAC &&\
   kubectl apply -f codewind-odoclusterrole.yaml &&\
   kubectl apply -f codewind-odoclusterrolebinding.yaml &&\
   cd - &&\
   rm -rf codewind-odo-extension
```
It will go to the correct location to add additional rules then do the cleanup for user